master/manifests/site.pp

```diff
@@ -160,68 +126,29 @@
 }
 
 # required by ghprb
-jenkins::plugin {
-  'git': ;
-}
-
-jenkins::plugin {
-  'git-client': ;
-}
-
+jenkins::plugin { 'git': ; }
+jenkins::plugin { 'git-client': ; }
 # required by ghprb
-jenkins::plugin {
-  'github': ;
-}
-
-jenkins::plugin {
-  'github-api': ;
-}
-
-jenkins::plugin {
-  'greenballs': ;
-}
-
-jenkins::plugin {
-  'groovy': ;
-}
-
-jenkins::plugin {
-  'groovy-postbuild': ;
-}
-
-jenkins::plugin {
-  'jobrequeue': ;
-}
-
+jenkins::plugin { 'github': ; }
+jenkins::plugin { 'github-api': ; }
+jenkins::plugin { 'greenballs': ; }
+jenkins::plugin { 'groovy': ; }
+jenkins::plugin { 'groovy-postbuild': ; }
```

Comment:
It's only a test dependency: https://github.com/jenkinsci/groovy-postbuild-plugin/blob/7df1063ef6e1bc786cff7326fec08305e067aaf4/pom.xml#L91

I removed it from #101 again.